lib/web/scripts/install.go

```diff
@@ -178,7 +180,22 @@ var legacyInstallScript string
 // or by unpacking the tarball. Its usage should be phased out in favor of the updater-based
 // installation script served by getUpdaterInstallScript.
 func getLegacyInstallScript(ctx context.Context, opts InstallScriptOptions) (string, error) {
-	return legacyInstallScript, nil
+	tunedScript := strings.Replace(legacyInstallScript, `TELEPORT_VERSION=""`, fmt.Sprintf(`TELEPORT_VERSION="%s"`, opts.TeleportVersion), 1)
+	if opts.TeleportFlavor == types.PackageNameEnt {
+		tunedScript = strings.Replace(tunedScript, `TELEPORT_SUFFIX=""`, `TELEPORT_SUFFIX="-ent"`, 1)
+	}
```

Comment:
*nit*: Not worth holding up this PR, but you might consider anchoring these replace operations to avoid replacing partial matches, e.g., `CUSTOM_TELEPORT_SUFFIX`